test_tf2/CMakeLists.txt

```diff
@@ -34,15 +33,19 @@ include_directories(${GTEST_INCLUDE_DIRS})
 ament_add_gtest(buffer_core_test test/buffer_core_test.cpp)
```

Comment:
shouldn't all this be under a `if (BUILD_TESTING)` condition?
Unrelated to this PR though

Reply:
Yeah, that's a good point.  It really should be.  We have kind of ignored it because we don't build binary packages, but it would be nicer for source users.  I'll open an issue about it.

Reply:
#278